### PR TITLE
Set up database in server

### PR DESCRIPTION
**Pull request by shinjam3**

### CODE REVIEW

File: server/.env [hunk 1]
```diff
@@ -1 +1,2 @@
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam"
+DB=<database URL>
```
Comment (dproc96) [inline server/.env]: You can remove this change as Lin's PR will remove the .env file anyway. Just message the Slack what we all should add to the env file

File: server/app.js [hunk 2]
```diff
@@ -11,6 +11,9 @@ const { json, urlencoded } = express;
 
 var app = express();
 
+// database setup
+require('./db')();
```
Comment (dproc96) [inline server/app.js:15]: Good idea to separate the setup code here to simplify app.js

**Review by dproc96 — APPROVED** [review]
I had one small change request but then this should be good to go!

File: server/.env (hunk 1 above)
Comment (shinjam3) [reply server/.env]: ok sounds good

**Pull request merged**